decidim-core/lib/decidim/rails.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# This is an override for:
+# https://github.com/rails/rails/blob/main/railties/lib/rails/all.rb
+
+# This file is needed to remove the sprockets dependency from Rails.
+
+require "rails"
+
+%w(
+  active_record/railtie
+  active_storage/engine
+  action_controller/railtie
+  action_view/railtie
+  action_mailer/railtie
+  active_job/railtie
+  action_cable/engine
```

Comment:
⚠️ Mmm, I think we should only require the railties used by Decidim, and let the developers to add the extra requires if they use them or replace them with alternative gems. AFAIK we only need `active_record`, `active_storage`, `action_controller`, `action_view`, `action_mailer` and `active_job`. What do you think?

Reply:
@leio10 `action_cable` is currently required at the [design app](https://github.com/decidim/decidim/blob/6d18f27b341220ca8692c1f267ec88f10d6e7737/decidim_app-design/config/application.rb#L13), although I don't think it is needed.

The [current documentation](https://github.com/decidim/decidim/blob/9f656b2e6a72cac8fb3e0fd1bd4121c0459c3d18/docs/modules/develop/pages/guide_migrate_webpacker_app.adoc#remove-sprockets-references) suggests adding `action_mailbox`, `action_cable` and `action_text` as well, although I don't think they are needed.

Regarding `rails/test_unit` I am not sure and I would expect it is needed. It is also [currently included](https://github.com/decidim/decidim/blob/b51773cb8c71d34b75af7c64a8769e5967e5a1c4/decidim-generators/lib/decidim/generators/install_generator.rb#L116), so I wouldn't remove it.

So, based on this, I removed `action_cable`, `action_mailbox` and `action_text`.

Reply:
Mmm, I'm not 100% sure, but I'd say that `rails/test_unit/railtie` is used to include Minitest, while Decidim and most of the Decidim apps are using RSpec, so I guess it won't be needed. In fact, I've removed it from the staging app and it's still working properly: https://github.com/codegram/decidim-staging/blob/main/config/application.rb.

Regarding `action_cable`, we can remove it from here and, if it's really needed in the design app, add it on it's `application.rb` file.

Reply:
OK, I think you are correct @leio10 after checking what [that module](https://github.com/rails/rails/blob/main/railties/lib/rails/test_unit/railtie.rb) does.

It seems to add some generators for the Minitest files + some rake tasks for running Minitests.

Therefore, I removed  `rails/test_unit` as well as requested.